src/renderer/src/components/TransactionActivityList/TransactionActivityListEvent.tsx

```diff
@@ -91,15 +100,17 @@ export const TransactionActivityListEvent = ({ event }: TProps) => {
           const tokenSymbol = token?.symbol ?? ''
           const tokenName = token?.name ?? ''
 
-          if (!tokenSymbol && !tokenName) return null
-
           return (
             <TransactionActivityListEventColumn
               label={t('columns.tokenLabel')}
               data={
-                <TransactionActivityListTooltip data={tokenName || tokenSymbol}>
-                  <span className="inline-block truncate max-w-28">{tokenSymbol || tokenName}</span>
-                </TransactionActivityListTooltip>
+                !tokenSymbol && !tokenName ? (
```

Comment:
How about creating a variable like `const hasToken = !!tokenSymbol && !!tokenName` and using it here?